particles/Tutorial/Kotlin/5_Collections/Collections.kt

```diff
@@ -0,0 +1,42 @@
+package arcs.tutorials
+
+import arcs.*
+import kotlin.native.internal.ExportForCppRuntime
+
+/**
+ * Sample Kotlin-WASM Particle to use a JSON store.
+ */
+class CollectionsParticle : Particle() {
+
+   private val people = Collection { CollectionsParticle_InputData() }
+   init {
+       registerHandle("inputData", people)
+   }
+
+    override fun populateModel(slotName: String, model: Map<String, String>): Map<String, Any> {
+        
+        val peopleList = ArrayList<Map<String, String>>()
+        people.forEach { people -> peopleList.add(mapOf("name" to people.name, "age" to people.age.toString())) }
+
+        return model + mapOf(
+            "people" to mapOf(
+                "\$template" to "person",
+                "models" to peopleList
+            )
+        )   
+    }
+
+    override fun getTemplate(slotName: String): String {
```

Comment:
Optional nit: this could be a function expression:

```kotlin
override fun getTemplate(slotName: String): String =
    """
    Hello to everyone:
    <ul>{{people}}</ul>
    
    <template person>
      <!-- ...... -->
    </template>
    """.trimIndent()
```